app/controllers/podcasts_controller.rb

```diff
@@ -8,10 +8,14 @@ def index
   end
 
   def show
-    @podcast = Podcast.find(params[:id])
-    @all_episodes = @podcast.episodes.sort_by { |e| e.release_date.to_date }.reverse
-    @episodes = @all_episodes.first(5)
-    @subscribers = @podcast.subscriptions.first(5)
+    if params[:id].present?
```

Comment:
it looks good. Would you like to add a test for it since we need to be sure when someone removes this line.

Reply:
I added two testcases for #show but can't be sure as bundle exec rspec is throwing errors